pkgs/by-name/fa/fastcap/package.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, fetchzip
+, ghostscript
+, lib
+, texlive
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastcap";
+  version = "2.0-18Sep92";
+
+  src = fetchzip {
+    url = "http://www.rle.mit.edu/cpg/codes/fastcap-${version}.tgz";
+    hash = "sha256-fnmC6WNd7xk8fphxkMZUq2+Qz+2mWIP2lvBUBAmUvHI";
+    stripRoot=false;
+  };
+
+  patches = [
+    ./patches/fastcap-mulGlobal-drop_conflicting_lib.patch
+    ./patches/fastcap-mulSetup-add_forward_declarations.patch
+  ];
+
+  nativeBuildInputs = [
+    ghostscript
+    texlive
+  ];
+
+  postPatch = ''
+    substituteInPlace ./doc/Makefile \
+      --replace '/bin/rm' 'rm'
+
+    for f in "doc/*.tex" ; do
+      sed -i -E 's/\\special\{psfile=([^,]*),.*scale=([#0-9.]*).*\}/\\includegraphics[scale=\2]{\1}/' $f
+      sed -i -E 's/\\psfig\{figure=([^,]*),.*width=([#0-9.]*in).*\}/\\includegraphics[width=\2]{\1}/' $f
+      sed -i -E 's/\\psfig\{figure=([^,]*),.*height=([#0-9.]*in).*\}/\\includegraphics[height=\2]{\1}/' $f
+      sed -i -E 's/\\psfig\{figure=([^,]*)\}/\\includegraphics{\1}/' $f
+    done
+
+    for f in "doc/mtt.tex" "doc/tcad.tex" "doc/ug.tex"; do
+      sed -i -E 's/^\\documentstyle\[(.*)\]\{(.*)\}/\\documentclass[\1]{\2}\n\\usepackage{graphicx}\n\\usepackage{robinspace}/' $f
+      sed -i -E 's/\\setlength\{\\footheight\}\{.*\}/%/' $f
+      sed -i -E 's/\\setstretch\{.*\}/%/' $f
+    done
+  '';
+
+  dontConfigure = true;
+
+  makeFlags = [
+    "CC=gcc"
+    "RM=rm"
+    "SHELL=sh"
+    "all"
+  ];
+
+  postBuild = ''
+    make manual
+    pushd doc
+    find -name '*.dvi' -exec dvipdf {} \;
```

Comment:
This seems to spit some warnings about missing fonts `Times-Roman`, `Symbol`, `Times-Roman`. Since these are just warnings and it generates successfully with computer modern, I'll leave it up to you if you'd like to include the fonts or not.